src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSGapicSurfaceTransformer.java

```diff
@@ -330,7 +329,8 @@ private ViewModel generateApiClass(GapicInterfaceContext context, boolean hasMul
               .fileHeader(
                   fileHeaderTransformer.generateFileHeader(
                       productConfig, ImportSectionView.newBuilder().build(), namer))
-              .packageName(packageMetadataNamer.getMetadataIdentifier());
+              .packageName(packageMetadataNamer.getMetadataIdentifier())
```

Comment:
Can you add a todo here to transform this into an acceptable name. Perceivably this name can have a hyphen in it like shown in the multiple-services baseline, which would cause an error.

Reply:
Filed #1615